currentYear

```diff
@@ -1 +1 @@
-2021
+2022
```

Comment:
The changes need to be propagated to `nextYear`, `previousYear`, as well as `en/*` too please

Reply:
Done.

I left the leap year thing alone but feel like we should add it in as a bank holiday.

Reply:
Oh, and happy new year!